packages/bierzo-wallet/src/logic/transactions/types/BwParserFactory.ts

```diff
@@ -234,6 +234,14 @@ export class BwParserFactory {
       return new BwRegisterUsernameParser();
     } else if (isUpdateTargetsOfUsernameTx(payload)) {
       return new BwUpdateUsernameTargetParser();
+    } else if (isRegisterAccountTx(payload)) {
+      return new BwUpdateUsernameTargetParser();
+    } else if (isReplaceAccountTargetsTx(payload)) {
+      return new BwUpdateUsernameTargetParser();
+    } else if (isTransferAccountTx(payload)) {
+      return new BwUpdateUsernameTargetParser();
+    } else if (isTransferDomainTx(payload)) {
+      return new BwUpdateUsernameTargetParser();
```

Comment:
Why this changes with `BwUpdateUsernameTargetParser`?

Reply:
Because otherwise it will return `BwUnkownParser` which is not right. It works but I am not sure that this is right behaviour.

Reply:
The parsers are on the `types` directory, each on its corresponding `Bw<Whatever>Tx`

Reply:
got it